docs/usage/private-modules.md

```diff
@@ -91,7 +91,7 @@ Here is an example of some host rules:
 Renovate applies theses `hostRules` to every HTTP(s) request which is sent, so they are largely independent of any platform or datasource logic.
 With `hostRules` in place, private package lookups should all work.
 
-## Release Notes
+## Looking up Release Notes
```

Comment:
We mention four times where Renovate needs credentials (including `- Looking up release notes`), so I think it makes sense to use the same words in the heading for this section.